com.unity.render-pipelines.universal/Shaders/ComplexLit.shader

```diff
@@ -469,10 +471,12 @@ Shader "Universal Render Pipeline/Complex Lit"
 
             HLSLPROGRAM
             #pragma only_renderers gles gles3 glcore
+            #pragma target 2.0
```

Comment:
So these subshaders in ComplexLit are just copypaste from Lit (600 lines). The plan was to remove these and use fallback to ease the maintenance. There are some comment blocks to mark the copypasted areas. A real Subshader for SM2.0 probably should be added after the main Subshader as Subshaders are picked in order.
There was some talk if we'd just add one for gles3/webgl otherwise fallback.
I suppose this isn't that fix and while we use this workaround, it is important keep it in sync, so that it actually works. So this is still the correct thing to do. Just to keep in mind that this code might be discarded in the future.

Reply:
Yes, this needs to be cleaned up when the fallback fix arrives. 

I'm noticing though that I'm not stripping Clear Coat from OpenGL in the ShaderPreProcessor. That should be done, right?